src/main/scala/Chisel/Tester.scala

```diff
@@ -1,39 +0,0 @@
-// See LICENSE for license details.
```

Comment:
Er so, can we get a Readme update as to where this is going?

Is chisel-testers 'released' now? (at the very least, needs a LICENSE)